app/views/dashboard/_node_meta.html.erb

```diff
@@ -4,11 +4,8 @@
 <%= distance_of_time_in_words(node.created_at, Time.current, false, :scope => :'datetime.time_ago_in_words') %>
 <% node = node.parent if node.is_a?(DrupalNodeRevision) %>
 <span class="">
-  <% if params[:controller] == 'questions' %>
```

Comment:
Hi, @prabhagharan -- this would, unfortunately, show answers for **all** nodes -- and this template is used for wiki pages, notes, etc etc. 

So I think we'd need to still keep the `if` clause -- but just below, you could leave the `else` clause out -- that way the comments would show for **both** questions and normal nodes. Make sense?